bindings/pyroot/pythonizations/python/ROOT/_pythonization/__init__.py

```diff
@@ -55,6 +55,258 @@ def pythonization(class_name, ns='::', is_prefix=False):
     Returns:
         function: function that receives the user-defined function and
             decorates it.
+
+    ### Pythonization example 
```

Comment:
I think these two tutorials make the docstring of this decorator far too long. What I would propose is to add the text to the newly created Python Interface documentation page instead, i.e. https://root.cern/doc/master/group__Pythonizations.html , found at `bindings/pyroot/pythonizations/doc`.

Reply:
I agree, I moved the text where you suggested. Thanks!